packages/blockchain-utils-linking/src/ethereum.ts

```diff
@@ -133,16 +133,16 @@ function safeSend(provider: any, method: string, params?: Array<any>): Promise<a
 
   if (provider.request) {
     return provider.request({ method, params }).then(
-      (response) => response,
-      (error) => {
+      (response: any) => response,
```

Comment:
Same as above, be nice if this had a type other than any.

Reply:
I just realised that `.then` call here is a no-op :/ so removed it.